src/main/java/com/mixpanel/android/mpmetrics/MixpanelAPI.java

```diff
@@ -1324,9 +1328,10 @@ public void onPrefsLoaded(SharedPreferences preferences) {
     }
 
     /* package */ UpdatesFromMixpanel constructUpdatesFromMixpanel(final Context context, final String token) {
-        if (Build.VERSION.SDK_INT < MPConfig.UI_FEATURES_MIN_API) {
-            Log.i(LOGTAG, "Web Configuration, A/B Testing, and Dynamic Tweaks are not supported on this Android OS Version");
-            return new UnsupportedUpdatesFromMixpanel(sSharedTweaks);
+        if (Build.VERSION.SDK_INT < MPConfig.UI_FEATURES_MIN_API
```

Comment:
Though what you said makes sense, I'd like to keep only one of them. Because both of them do exactly the same thing - 1. I don't want to confuse our users by having two different classes but both have the same implement 2. we have duplicated code.

My take is that their names do have different meanings, but the end results are the same - both of them won't initialize ViewCrawler initialization. So, I'd like to keep only `NoOpUpdatesFromMixpanel`.